spec/system/candidate_interface/entering_details/candidate_entering_other_qualification_spec.rb

```diff
@@ -38,14 +38,14 @@
     and_i_choose_not_to_add_additional_qualifications
     and_click_save_and_continue
     then_i_see_the_other_qualification_review_page
-    and_i_should_see_my_qualifications
+    and_i_see_my_qualifications
     and_my_other_uk_qualification_has_the_correct_format
 
     when_i_select_add_another_qualification
     and_choose_as_level
     then_the_form_is_empty
     and_i_visit_the_other_qualification_review_page
-    then_i_should_not_see_an_incomplete_as_level_qualification
+    then_i_not_see_an_incomplete_as_level_qualification
```

Comment:
The wording on these descriptions is a bit wonky after removing the word "should"